aws/resource_aws_msk_cluster_test.go

```diff
@@ -515,6 +515,44 @@ func TestAccAWSMskCluster_LoggingInfo(t *testing.T) {
 	})
 }
 
+func TestAccAWSMskCluster_KafkaVersionUpdate(t *testing.T) {
```

Comment:
Since version update is separate from version update with config change, we should have a 2nd acceptance test added for version update with config change.

Reply:
Added `TestAccAWSMskCluster_KafkaVersionUpdateWithConfigurationInfo` to cover this